android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/Project.java

```diff
@@ -163,20 +163,7 @@ public void writeToParcel(Parcel dest, int flags) {
         dest.writeDouble(project_files_downloaded_time);
         dest.writeDouble(last_rpc_time);
 
-        dest.writeBooleanArray(new boolean[]{
-                master_url_fetch_pending,
-                non_cpu_intensive,
-                suspended_via_gui,
-                dont_request_more_work,
-                scheduler_rpc_in_progress,
-                attached_via_acct_mgr,
-                detach_when_done,
-                ended,
-                trickle_up_pending,
-                no_cpu_pref,
-                no_cuda_pref,
-                no_ati_pref
-        });
+        dest.writeBooleanArray(new boolean[]{master_url_fetch_pending, non_cpu_intensive, suspended_via_gui, dont_request_more_work, scheduler_rpc_in_progress, attached_via_acct_mgr, detach_when_done, ended, trickle_up_pending, no_cpu_pref, no_cuda_pref, no_ati_pref});
```

Comment:
This is a type of change that makes things **clearly** worse. The human editor had clear and justified intentions (with the line breaks) which should be preserved.